Common/ExecutionModel/vtkThreadedImageAlgorithm.cxx

```diff
@@ -333,4 +448,3 @@ void vtkThreadedImageAlgorithm::ThreadedExecute(
   (void)threadId;
   vtkErrorMacro("Subclass should override this method!!!");
 }
```

Comment:
Not sure why it displays the last line as missing.  When I go into the file to modify it, it shows the last line. I will investigate why git is showing this.